drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -179,20 +179,57 @@ function DragNDropTemplates(url_name) {
     var feedbackTemplate = function(ctx) {
         var feedback_display = ctx.feedback_html ? 'block' : 'none';
         var reset_button_display = ctx.display_reset_button ? 'block' : 'none';
-        var properties = { attributes: { 'aria-live': 'polite' } };
         return (
-            h('section.feedback', properties, [
+            h('section.feedback', {}, [
                 h(
                     'button.reset-button.unbutton.link-button',
-                    { style: { display: reset_button_display }, attributes: { tabindex: 0 }, 'aria-live': 'off'},
+                    { style: { display: reset_button_display }, attributes: { tabindex: 0, 'aria-live': 'off' } },
```

Comment:
Nice catch. I tested whether moving `arial-live="polite"` back to the `section.feedback` container makes any difference with this change, but it looks like it doesn't. VoiceOver still reads out "Reset problem" the first time it appears, even though `aria-live="off"` is set correctly, so moving the button out of `aria-live="polite"` aria as you did here still makes sense.